### ISSUE TITLE
Awesome Plugin...

### ISSUE THREAD

**Issue by prog20901** [hhurz/tableExport.jquery.plugin#194]
This is an awesome plugin..I saw your comment in one of the issue...forked from an abaondoned plugin and modified it...Great...This would be a great plugin if all the existing issues has been addressed and couple of new improvement are done...I see there are lots of contributors as well..

Hope all will be resolved and can see some improvements as well.

**Comment by schutze** [comment]
I agree.  The other HTML export plugins I have seen are not nearly as reliable and full featured.  It would be nice if this could be refined and formalized into a "packaged" solution.

**Comment by prog20901** [comment]
@hhurz Do you have any updates on this request?

**Comment by hhurz** [comment]
Nope. This is a leisure project. I just have enough time to keep the project alive. Thanks.

**Comment by prog20901** [comment]
hhurz any updates on the request?

**Comment by hhurz** [comment]
@jkathir what exactly was your request? Any details?

**Comment by prog20901** [comment]
i remember you have agreed to have couple of examples folder which has HTML files which has all export options with table with dummy values. Similar couple of example HTML files in separate folder as Example1, Example2, Example3, Example4, Example5. One example with complete features in a single HTML file with a table to view all in one shot. a sample txt file as readme in each folder to understand what it does. It would take an hour for you but it would save a huge time for anyone who downloads this .
